labs/lab-1-html-and-css/style-pt-2.css

```diff
@@ -0,0 +1,158 @@
+body {
+  background-color: #f9fafc;
+  font-family: 'Open Sans', sans-serif;
+  padding: 50px;
+}
+
+h1 {
+  font-weight: normal;
+  font-size: 40px;
+}
+
+p {
+  color: #69696a;
+}
+
+.logo {
+  text-align: left;
+  position: absolute;
+  top: 30px;
+}
+
+nav {
+  text-align: left;
+  font-weight: normal;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav {
+  padding-top: 15px;
+  color: #69696a;
+  text-decoration: none;
+}
+
+.button {
+  background-color: black;
+  color: #f9fafc;
+  padding: 10px 30px;
+  border-radius: 50px;
+  width: auto;
+  margin-top: 20px;
+  text-align: center;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
+.intro {
+  position: relative;
+  z-index: 1;
+}
+
+.intro-copy {
+  padding-left: 30px;
+  border-left: solid 1px #69696a;
+}
+
+.circles {
+  display: none;
+}
+
+.scroll-button {
+  background-color: black;
+  color: #f9fafc;
+  padding: 10px 30px;
+  border-radius: 50px;
+  width: auto;
+  text-align: center;
+  position: fixed;
+  bottom: 50px;
+  left: 50%;
+  margin-left: -25px;
+  z-index: 1;
+  /* margin-left: auto;
+  margin-right: auto; ----- could not get scroll button to be exactly centered, landed on left: 50%; margin-left: -25px; being the closest */
```

Comment:
Yes, it seems no one quite landed on the trick of this -- but you were on the right track, and wise of you to comment about the problem you were having.

I mean, you could have asked about it on slack...

Next time, try nesting your button with `margin: auto` *inside* a div with `position: fixed`. Can't do both at the same time on the same element.

Good on you for thinking of using `fixed` instead of `absolute`, though! You're getting inside the designer's head, thinking about their intentions.